packages/components/src/internal/renderers.tsx

```diff
@@ -56,9 +125,15 @@ export const HeaderCellDropdown: FC<HeaderCellDropdownProps> = memo(props => {
     const allowColumnViewChange = (handleHideColumn || handleAddColumn) && !!model;
     const includeDropdown = allowColSort || allowColFilter || allowColumnViewChange;
 
+    useEffect(() => {
+        return () => {
+            setOpen(false);
+        }
+    }, []);
```

Comment:
could this just be `useEffect(() => setOpen(false), []);` or is there a reason for the anonymous function?

Reply:
This is, I believe, how one gets something equivalent to `componentWillUnmount`. The menu was staying open unnecessarily.

Reply:
ah yes, that makes sense.